test/typography-preload.js

```diff
@@ -0,0 +1,15 @@
+import { fonts, importUrl } from '../components/typography/typography.js';
+
+if (!document.head.querySelector('.d2l-typography-font-face-preload')) {
+	for (const type of Object.keys(fonts)) {
+
+		const preload = document.createElement('link');
+		preload.rel = 'preload';
+		preload.className = 'd2l-typography-font-face-preload';
+		preload.href = `${new URL(`${fonts[type]}.woff2`, importUrl)}`;
+		preload.as = 'font';
+		preload.crossOrigin = 'anonymous';
```

Comment:
This reminds me... we should probably do this in the LMS. Separate issue though.